content/python/concepts/random-module/terms/seed/seed.md

```diff
@@ -15,46 +15,44 @@ CatalogContent:
   - 'paths/data-science'
 ---
 
-Save the state of the pseudo-random number generator.
+In the Python `random` module, the `.seed()` function is used to create a pseudo-random number generator. Pseudo-random number generators appear to produce random numbers by performing some operation on a value. This value is the seed, which acts as the first "previous" value in the sequence of numbers. By using a particular seed value, a user is able to reproduce the same random numbers multiple times.
 
 ## Syntax
 
-```py
-random.seed()
+The `.seed()` function sets the first random number of the generator, either with a `value` or without one:
+
+```pseudo
+random.seed(value)
 ```
 
-Pseudo-random number generators appear to produce random numbers by performing some operation on a value. This value is the seed, which acts as the first "previous" value in the sequence of numbers. By using a particular seed value, a user is able to reproduce the same random numbers multiple times. If no value is input into the method, the random number generator will use the current system time to create the seed.
+If a `value` is not provied, the random number generator will use the current system time to create the seed. The `value` can be an `int`, `float`, `byte`, `bytearray`, or `NoneType`, and
```

Comment:
Was something supposed to be added after the ", and"?

Reply:
Haha no I don't think so, I'll fix that.